library/Faktory/Job.hs

```diff
@@ -101,6 +103,16 @@ jobRetriesRemaining job = max 0 $ enqueuedRetry - attemptCount
   enqueuedRetry = maybe faktoryDefaultRetry getLast $ joRetry $ jobOptions job
   attemptCount = maybe 0 ((+ 1) . jfRetryCount) $ jobFailure job
 
+jobReserveForMicroseconds :: Job arg -> Int
+jobReserveForMicroseconds =
+  (* 1000)
```

Comment:
If `reserve_for` is in seconds, wouldn't `* 1000` be milliseconds? I think you'd want * 1,000,000 for microseconds.

I'm guessing you want this to be directly passable to `timeout`,

> in case no result is available within n microseconds (1/ **10^6** seconds)

Reply:
I literally always screw this up :laughing: